docs/developers/development-env.md

```diff
@@ -93,25 +93,25 @@ You will need two windows
 - To run in debug mode set log level DEBUG before running
 
   ```
-  export ARGO_CLOUDOPS_LOG_LEVEL=DEBUG
+  export CELLO_LOG_LEVEL=DEBUG
   make ; make up
   ```
 
 ### Create Cello Project And Target (One Time Setup)
 
-- In window **#2**, ensure you have the **ARGO_CLOUDOPS_ADMIN_SECRET**
+- In window **#2**, ensure you have the **CELLO_ADMIN_SECRET**
   env set to the same value used above.
 
 - Ensure your credentials are set for the **target account** and create your first
-  project and target. This returns the **ARGO_CLOUDOPS_USER_TOKEN** for the new project.
+  project and target. This returns the **CELLO_USER_TOKEN** for the new project.
```

Comment:
I don't think user token should be updated to use `CELLO` for the prefix. I think that one is being read by the cli as `ARGO_CLOUDOPS_USER_TOKEN` still